### DISCUSSION TITLE
About Stripe API version

### DISCUSSION THREAD

**Discussion by HashNuke** [pay-rails/pay#1050] [General]
Hello excid3 and all,
I searched the repo and found only one place where the Stripe API version is mentioned - that is in the changelog.
The Stripe version is mentioned as `2023-08-16` for v6.8.

Our new Stripe account defaults to `2024-06-20` according to Stripe dashboard. Should we be setting Stripe's API version `Stripe.api_version='2023-08-16'` when using `pay`?

**Info that could be documented:**
The Stripe webhook endpoint creation form displays a select input for API version. The Stripe integration docs in could highlight the API version to ensure it matches with the gem's requirements. I can send a PR to update this info in the docs.

**Comment by excid3** [comment]
Stripe API versions are controlled by their gem now. https://github.com/stripe/stripe-ruby/blob/master/CHANGELOG.md#1200---2024-06-24
